MockerTests/MockTests.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  MockTests.swift
+//  
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: MockerTests/MockTests.swift;
  Line: 3;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)